src/typedefs.rs

```diff
@@ -533,6 +548,21 @@ macro_rules! construct_tag {
         /// # Security:
         /// - __**Avoid using**__ `unprotected_as_bytes()` whenever possible, as it breaks all protections
         /// that the type implements.
+        ///
+        /// - The trait `PartialEq<&'_ [u8]>` is implemented for this type so that users are not tempted
+        /// to call `unprotected_as_bytes` to compare this sensitive value to a byte slice. The trait
+        /// is implemented in such a way that the comparison happens in constant time. Thus, users should
+        /// prefer `SecretType == &[u8]` over `SecretType.unprotected_as_bytes() == &[u8]`.
+        /// Examples are shown below. The examples apply to any type that implements `PartialEq<&'_ [u8]>`.
+        /// ```rust
+        /// use orion::hazardous::mac::hmac::Tag;
+        ///
+        /// // Initialize an arbitrary, 64-byte tag.
+        /// let tag = Tag::from_slice(&[1; 64]).unwrap();
+        ///
+        /// // Constant-time comparison
```

Comment:
Please make this comment consistent with the others, i.e change to "Secure, costant-time comparison with byte slice".